### PR TITLE
Blacklist properties

### PR DESCRIPTION
**Pull request by dspautz**
Please review my additional code to to support blacklisted properties for PropertySpelling.
Let me know if something is wrong.

Thank you and have a nice day

### CODE REVIEW

File: lib/scss_lint/linter/property_spelling.rb [hunk 1]
```diff
@@ -9,7 +9,9 @@ class Linter::PropertySpelling < Linter
                            .to_set
 
     def visit_root(_node)
-      @extra_properties = config['extra_properties'].to_set
+      @extra_properties = config['extra_properties'] ? config['extra_properties'].to_set : []
+      @disabled_properties = config['disabled_properties'] ? config['disabled_properties'].to_set : []
```
Comment (sds) [inline lib/scss_lint/linter/property_spelling.rb]: Can write as:

```ruby
@extra_properties = Array(config['extra_properties']).to_set
@disabled_properties = Array(config['disabled_properties']).to_set
```

File: .scss-lint.yml [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+    properties: [background-color, border-color, color]
+
+  VendorPrefix:
+    enabled: false
+
+  ZeroUnit:
+    enabled: true
+
+  Compass::*:
+    enabled: true
```
Comment (sds) [inline .scss-lint.yml]: This file should be removed.

File: spec/scss_lint/linter/property_spelling_spec.rb [hunk 3]
```diff
@@ -55,6 +55,31 @@
     end
   end
 
+  context 'when disabled properties are specified' do
+    let(:linter_config) { { 'disabled_properties' => ['made-up-property'] } }
+
+
```
Comment (sds) [inline spec/scss_lint/linter/property_spelling_spec.rb]: Lose the extra line

**Comment by sds** [comment]
Thanks for the pull request, @dspautz.

It seems like the purpose of this change is to report blacklisted properties separately. Right now a blacklisted property will report "Unknown property ...". That doesn't seem correct. Instead, we should report a different message in the case the property is blacklisted, e.g. "Property '_property_' is prohibited."

**Comment by dspautz** [comment]
Please checkout my changes. Let me know when I can help.

**Comment by sds** [comment]
Thanks for address those comments, @dspautz.

Merged in f165d94.

**Pull request closed without merging**